_studio/mfx_lib/vpp/src/mfx_vpp_hw.cpp

```diff
@@ -2050,6 +2050,11 @@ mfxStatus VideoVPPHW::GetVideoParams(mfxVideoParam *par) const
             MFX_CHECK_NULL_PTR1(bufFRC);
             bufFRC->Algorithm = m_executeParams.frcModeOrig;
         }
+        else if (MFX_EXTBUFF_VPP_COLORFILL == bufferId)
+        {
+            mfxExtVPPColorFill *bufColorfill = reinterpret_cast<mfxExtVPPColorFill *>(par->ExtParam[i]);
+            MFX_CHECK_NULL_PTR1(bufColorfill);
```

Comment:
I think you are supposed to assign currently configured values for the color fill. Idea is that user calls GetVideoParams and sees how VPP is configured. Please, add assignment.

Reply:
+1

Reply:
done